clients/1.0.0/ruby/a15k_client.gemspec

```diff
@@ -24,12 +24,12 @@ Gem::Specification.new do |s|
   s.homepage    = "https://github.com/swagger-api/swagger-codegen"
   s.summary     = "Assessment Network API Ruby Gem"
   s.description = "> Stores content for the Assessment Network  API Version is specified in the URL path as `api/<version>/resource` Version will default to the latest if it's ommited  All endpoints require an API key to be passed in the request header. API keys can be obtained by members at www.a15k.org. "
-  # TODO uncommnet and update below with a proper license
+  # TODO uncommnet and update below with a proper license 
   #s.license     = "Apache 2.0"
   s.required_ruby_version = ">= 1.9"
 
   s.add_runtime_dependency 'typhoeus', '~> 1.0', '>= 1.0.1'
-  s.add_runtime_dependency 'json'
+  s.add_runtime_dependency 'json', '~> 2.1', '>= 2.1.0'
 
```

Comment:
I had to manually edit the generated client lib's JSON specified version so it could be used in exercises.  I can't recall the details but I think there was a conflict with using json 2.1 with rails 4.1 or something like that.  

You might try to bundle update the a15k_client lib in exercises and see if you hit that.

Reply:
Ah I had been doing that but I must have forgotten to do so on one of the later runs!